packages/rum/src/boot/recorderApi.ts

```diff
@@ -155,3 +155,7 @@ export function makeRecorderApi(
     isRecording: () => state.status === RecorderStatus.Started,
   }
 }
+
+function isBrowserSupported() {
+  return typeof Array.from === 'function' && typeof CSSSupportsRule === 'function'
+}
```

Comment:
**Nitpick**: Would you please add a comment to explain the reason behind identifying browser support from the support of `Array.from` and `CSSSupportsRule` ?

Reply:
Done